Dockerfile.slim

```diff
@@ -1,12 +1,15 @@
 FROM alpine:3.11
 LABEL maintainer="sig-platform@spinnaker.io"
 
+ENV KUSTOMIZE_VERSION=3.8.1
```

Comment:
I'd HIGHLY recommend 3.8.5 - hit a bug think in 3.8.1 ( think another PR on this would have to go look)

Reply:
Thanks Jason, added commit with 3.8.5 per your PR #733.
I can drop the extra commit or we can merge this and close yours given I've also changed Dockerfile. Whatever works for everyone to get this across the line. :)

Kustomize looks to be up to 3.8.9 patch version now but I'm not familiar enough with the project to understand the changelog so probably safest to roll with 3.8.5 as you suggest. Happy to amend if someone requests though.